lib/fluent/plugin/in_prometheus.rb

```diff
@@ -54,6 +58,8 @@ def configure(conf)
 
       @base_port = @port
       @port += fluentd_worker_id
+
+      raise "Invalid content encoding for the exposed metrics endpoint" unless @content_encoding="identity" || @content_encoding="gzip"
```

Comment:
what do you mean identity?

Reply:
https://devdoc.net/web/developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding.html
Basically its same as no compression